sonic_platform_base/sonic_sfp/sfputilbase.py

```diff
@@ -995,7 +997,27 @@ def get_transceiver_info_dict(self, port_num):
                 transceiver_info_dict['specification_compliance'] = str(compliance_code_dict)
 
                 transceiver_info_dict['nominal_bit_rate'] = str(sfp_interface_bulk_data['data']['NominalSignallingRate(UnitsOf100Mbd)']['value'])
-
+        sys.path.append('/usr/share/sonic/platform')
```

Comment:
this path only work inside PMON, is there some case that this file could be called from the host side?